website/docs/providers/type/community-index.html.markdown

```diff
@@ -17,52 +17,55 @@ please fill out this [community providers form](https://docs.google.com/forms/d/
-
-[Kibana](https://github.com/ewilde/terraform-provider-kibana)
-
-[Kong](https://github.com/kevholditch/terraform-provider-kong)
-
-[LXD](https://github.com/sl1pm4t/terraform-provider-lxd)
-
-[Sentry](https://github.com/jianyuan/terraform-provider-sentry)
-
-[Microsoft SCVMM](https://github.com/GSLabDev/terraform-provider-scvmm)
-
-[Open Day Light](https://github.com/GSLabDev/terraform-provider-odl)
-
-[Pass](https://github.com/camptocamp/terraform-provider-pass)
-
-[Puppet CA](https://github.com/camptocamp/terraform-provider-puppetca)
-
-[PuppetDB](https://github.com/camptocamp/terraform-provider-puppetdb)
-
-[Runscope](https://github.com/ewilde/terraform-provider-runscope)
+<table class="table">
+    <td><a href="https://github.com/vancluever/terraform-provider-acme">ACME/Let's Encrypt</a></td>
+    <td><a href="https://github.com/GSLabDev/terraform-provider-ad">Active Directory</a></td>
+    <td><a href="https://github.com/zambien/terraform-provider-apigee">Apigee</a></td>
+    </tr>
+    <tr>
+    <td><a href="https://github.com/avinetworks/terraform-provider-avi">AVI</a></td>
+    <td><a href="https://github.com/AviatrixSystems/terraform-provider-aviatrix">Aviatrix</a></td>
+    <td><a href="https://github.com/nicolai86/terraform-provider-couchdb">CouchDB</a></td>
+    </tr>
+    <tr>
+    <td><a href="https://github.com/rackn/terraform-provider-drp/">Digital Rebar</a></td>
+    <td><a href="https://github.com/tiramiseb/terraform-provider-gandi">Gandi</a></td>
+    <td><a href="https://github.com/drewsonne/terraform-provider-gocd">GoCD</a></td>
+    </tr>
+    <tr>
+    <td><a href="https://github.com/sethvargo/terraform-provider-googlecalendar">Google Calendar</a></td>
+    <td><a href="https://github.com/mcuadros/terraform-provider-helm">Helm</a></td>
+    <td><a href="https://github.com/hetznercloud/terraform-provider-hcloud">Hetzner Cloud</a></td>
+    </tr>
+    <tr>
+    <td><a href="https://github.com/GSLabDev/terraform-provider-httpfileupload">HTTP File Upload</a></td>
+    <td><a href="https://github.com/HewlettPackard/terraform-provider-oneview">HP OneView</a></td>
+    <td><a href="https://github.com/sky-uk/terraform-provider-infoblox">Infoblox</a></td>
+    </tr>
+    <tr>
+    <td><a href="https://github.com/anubhavmishra/terraform-provider-jira">Jira</a></td>
+    <td><a href="https://github.com/Mongey/terraform-provider-kafka">Kafka</a></td>
+    <td><a href="https://github.com/ewilde/terraform-provider-kibana">Kibana</a></td>
+    </tr>
+    <tr>
+    <td><a href="https://github.com/kevholditch/terraform-provider-kong">Kong</a></td>
+    <td><a href="https://github.com/sl1pm4t/terraform-provider-lxd">LXD</a></td>
+    <td><a href="https://github.com/sacloud/terraform-provider-sakuracloud">SakuraCloud</a></td>
+    </tr>
+    <tr>
+    <td><a href="https://github.com/jianyuan/terraform-provider-sentry">Sentry</a></td>
+    <td><a href="https://github.com/Yelp/terraform-provider-signalform">SignalFx</a></td>
```

Comment:
if we're ordering these alphabetically by product name, shouldn't this be below Runscope?